deploy/charts/external-secrets/values.yaml

```diff
@@ -338,3 +339,6 @@ certController:
       # requests:
       #   cpu: 10m
       #   memory: 32Mi
+
+# -- Specifies `dnsOptions` to deployment
+dnsConfig: {}
```

Comment:
```suggestion
dnsConfig: {}

```

Reply:
There's a linting error, a newline seems to be missing on the end of this file.

Reply:
done